functionary/builder/api/v1/serializers/package_definition.py

```diff
@@ -1,19 +1,10 @@
 """ Serializers for defining a package """
 from rest_framework import serializers
 
+from core.utils.parameter import PARAMETER_TYPE_CHOICES
+
 LANGUAGES = [("python", "Python"), ("javascript", "JavaScript")]
-PARAMETER_TYPES = [
-    ("integer", "Integer"),
-    ("string", "String"),
-    ("text", "String (Long)"),
-    ("float", "Float"),
-    ("file", "File"),
-    ("boolean", "Boolean"),
-    ("date", "Date"),
-    ("datetime", "Date Time"),
-    ("json", "JSON"),
-]
-RETURN_TYPES = PARAMETER_TYPES[:]
+RETURN_TYPE_CHOICES = PARAMETER_TYPE_CHOICES[:]
```

Comment:
I'm not entirely clear on why this is a separate thing from `PARAMETER_TYPE_CHOICES`, but that's how it was before and I decided to just keep it (though I renamed it for consistency).